README.md

```diff
@@ -354,6 +355,26 @@ _Mailing lists servers and mass mailing software - one message to many recipient
 - [SquirrelMail](http://squirrelmail.org) - Another browser-based IMAP client. ([Source Code](https://sourceforge.net/p/squirrelmail/code/HEAD/tree/)) `GPL-2.0` `PHP`
 - [WebMail Lite](http://www.afterlogic.org/webmail-lite) - Web-based IMAP Mail client. ([Source Code](https://github.com/afterlogic/webmail-lite)) `GPL-3.0` `PHP`
 
+#### Disposable email and forwarding services
+
+**[`^        back to top        ^`](#)**
+
+- [Ad Hoc Email Server (AHEM)](http://www.ahem.email) - An ad-hoc mail server. Create and use ad hoc disposable mailboxes for testing or any other purpose. http://www.ahem.email ([Demo](http://www.ahem.email/account/bi1vx86r), [Source Code](https://github.com/o4oren/Ad-Hoc-Email-Server)) `Angular/Nodejs` `MongoDB` `MIT`
```

Comment:
The demo link should probably be deleted to avoid overuse of that random ID. Not that big of a deal, but the main page (already linked) is instructive enough. If people wanted to use the hosted version it's easy enough to get started.

Reply:
fixed